svc-events/events/publishevttosubscriber_test.go

```diff
@@ -233,6 +254,17 @@ func TestPublishEventsToDestiantion(t *testing.T) {
 		flag := PublishEventsToDestination(event)
 		assert.True(t, flag)
 	}
+	for _, v := range messages {
+		var event common.Events
+		event.IP = "10.24.1.9"
```

Comment:
give some random IP. Please change in other places also.

Reply:
all other ut in  event service  using same kind ip we will update  same later